pbft/pbft_operation.cpp

```diff
@@ -115,3 +115,14 @@ pbft_operation::request_hash(const pbft_request& req)
     return req.ShortDebugString();
 }
 
+void
+pbft_operation::set_session(std::shared_ptr<bzn::session_base> session)
```

Comment:
reference

Reply:
I think move semantics are clear here; this function is obviously going to store a copy of its argument